### ISSUE TITLE
Editor axe violation on a smaller screen 

### ISSUE THREAD

**Issue by sudo-may** [NationalSecurityAgency/skills-service#2723]
"Ensures buttons have discernible text": 
![image](https://github.com/user-attachments/assets/df5026e3-a89c-4b1d-81fc-181914367538)

**Comment by sudo-may** [comment]
please note that the issue only occurs when a markdown editor is loaded with the viewport width > 950 and then resized down, the more button appears but `aria-label` is not added

**Comment by rmmayo** [comment]
:+1:
